src/components/accessories/billTable/styles.scss

```diff
@@ -0,0 +1,22 @@
+
+.bill_details {  
+    ul {
+        white-space: nowrap;   
+        overflow-x: hidden; 
+        margin-left: 0; 
+        padding-left: 0;
+    }
+    li span .MuiTypography-h5 {
+        padding-left: 0;
+        margin-left: 0;
+        display: inline;
+        text-decoration: none;    
+        color: #000; /* Fallback for older browsers */
+        color: rgba(0, 0, 0, 0.7);
```

Comment:
If in 2021 there is someone that still use IE 9 he / she didn't deserves this fallback :)

In every case, if you need something like that you could use some RGB color as for eg: #333232

Reply:
OK, I applied it